spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerTemplate.java

```diff
@@ -92,6 +93,23 @@ public SpannerMappingContext getMappingContext() {
 		return this.mappingContext;
 	}
 
+	public SpannerSqlTemplate getSpannerSqlTemplate() {
+		return new SpannerSqlTemplate() {
```

Comment:
Return a `spannerSqlTemplate` member instead. See my comment above.

Reply:
Why can't `SpannerTemplate` simply call `this.spannerSqlTemplate.executeQuery`?

Reply:
because that method would be private right? Isn't that the whole point?